### PR TITLE
Website Redesign: Product Page Enhancements

### PR DESCRIPTION
**Pull request by JasonMun7**
### Summary 
This PR addresses the following changes for the products page

- [x] Utilized reduce Motion to de-activate the animations within the page including the hover/ line fill animation
- [x] Reduced the overall sticker border for a more cleaner/sleeker look 
- [x] Made Icons of each display a little bigger and Increased space between the View Product and text 
- [x] Fixed overflow issue with images and decreased line usage for descriptions
- [x] Adjusted width at Have any ideas section 

### Notion/Figma Link 
This was updated after talking with @Bookie0!!
[Reduce Motion](https://www.notion.so/New-Website-PRODUCT-Add-RE-TO-Produce-Animations-Updated-14a0ad723ce1801c983bf1d5d7f7ba17?pvs=4)
[Replace logos](https://www.notion.so/New-Website-PRODUCTS-Logo-carousel-in-hero-section-14a0ad723ce180ecb67bcf46ece73bbb?pvs=4)

### Test Plan 
- See svg differences below

Before sizing and spacing: 
Screenshot 2024-11-29 at 12 47 58 AM
Screenshot 2024-11-29 at 12 48 32 AM
After Sizing and spacing: 
Screenshot 2024-11-29 at 12 46 55 AM
Screenshot 2024-11-29 at 12 27 18 AM

### Notes 
- Might need to reach out to Carriage to see if they can send us any designs we can use to update else we can just put cornell go back to the coming soon screen we had before 
- Added an Image classname prop to `ProductDisplay` component since each image in the figma has different width limits

### CODE REVIEW

**Comment by andrew032011** [comment]
> Might need to reach out to Carriage to see if they can send us any designs we can use to update else we can just put cornell go back to the coming soon screen we had before

Oh is this CornellGo or Carriage? If we need something feel free to tag the PM in of their channels and cc me + PM’s + designers.

**Comment by JasonMun7** [comment]
> > Might need to reach out to Carriage to see if they can send us any designs we can use to update else we can just put cornell go back to the coming soon screen we had before
> 
> Oh is this CornellGo or Carriage? If we need something feel free to tag the PM in one of their channels and cc me + PM’s + designers.

This is for Carriage. Right now Carriage has the Coming Soon/blurred image as a place holder and Clement suggested that we should keep it consistent and maybe revert the Cornell Go back to it's blurred image instead of the components we have right now. I was thinking if we could get some images of their features we could switch out the blurred coming soon with those new features (After we've structured it). Also again @Bookie0 feel free to add on/modify this and if either you @andrew032011 or @cchrischen! Have any ideas! 

Also that sounds like a great Idea!

**Review by andrew032011 — APPROVED** [review]
Screenshot 2024-11-29 at 8 41 11 AM
Screenshot 2024-11-29 at 8 41 05 AM
Screenshot 2024-11-29 at 8 41 02 AM
Noticed that a few of these icons aren't aligned with the product name? It looks like they were slightly off in the original too, but if it's easy to adjust that'd be nice!

**Comment by andrew032011** [comment]
> > > Might need to reach out to Carriage to see if they can send us any designs we can use to update else we can just put cornell go back to the coming soon screen we had before
> > 
> > 
> > Oh is this CornellGo or Carriage? If we need something feel free to tag the PM in one of their channels and cc me + PM’s + designers.
> 
> This is for Carriage. Right now Carriage has the Coming Soon/blurred image as a place holder and Clement suggested that we should keep it consistent and maybe revert the Cornell Go back to it's blurred image instead of the components we have right now. I was thinking if we could get some images of their features we could switch out the blurred coming soon with those new features (After we've structured it). Also again @Bookie0 feel free to add on/modify this and if either you @andrew032011 or @cchrischen! Have any ideas!
> 
> Also that sounds like a great Idea!

Reached out on Slack. https://cornelldti.slack.com/archives/CJCDQKR0X/p1732888246400639

File: new-dti-website/components/products/FloatingImages.tsx [hunk 1]
```diff
@@ -56,8 +59,8 @@ export default function FloatingImages({ images }: FloatingImagesProps) {
           <div
             key={index}
             className={`${index === 0 ? 'relative z-10' : 'absolute'} w-full h-auto ${
-              index !== 0 ? getFloatClass(img.direction) : ''
-            } ${img.delay && index !== 0 ? 'floating-delay' : ''}`}
+              index !== 0 && !reduceMotion ? getFloatClass(img.direction) : ''
+            } ${!reduceMotion && img.delay && index !== 0 ? 'floating-delay' : ''}`}
```
Comment (Bookie0) [inline new-dti-website/components/products/FloatingImages.tsx:63]: hmm for me it seems like the images still hover slightly even when with Reduce Motion settings on?

the lines don't animate which is expected but i still see the images move around a bit

https://github.com/user-attachments/assets/a4d7206d-d135-4d1c-8a20-1379b6549889

File: new-dti-website/public/icons/qmi_icon.svg
Comment (Bookie0) [inline new-dti-website/public/icons/qmi_icon.svg:1]: hmm ok so for the sticker effect, James, Vannessa, and I were discussing in [this thread here](https://www.figma.com/design/ttAGEX3pHmzuhMzp8uAyhz?node-id=1-30#1038252636) about the sticker effect. I think we all agree that thinner in general is the good direction but we haven't aligned yet on how thin to reduce the stroke.

This is def an improvement, so we can go ahead with your PR, but just wanted to let you know that we might modify it a bit more too.

**Review by Bookie0 — CHANGES_REQUESTED** [review]
thanks looking good!

just a little problem with the images in the Product page that keep animating even with Reduce Motion off

File: new-dti-website/components/products/FloatingImages.tsx (hunk 1 above)
Comment (JasonMun7) [reply new-dti-website/components/products/FloatingImages.tsx:63]: Will look into this! Thanks for catching that!

**Comment by JasonMun7** [comment]
> > > Might need to reach out to Carriage to see if they can send us any designs we can use to update else we can just put cornell go back to the coming soon screen we had before
> > 
> > 
> > Oh is this CornellGo or Carriage? If we need something feel free to tag the PM in one of their channels and cc me + PM’s + designers.
> 
> This is for Carriage. Right now Carriage has the Coming Soon/blurred image as a place holder and Clement suggested that we should keep it consistent and maybe revert the Cornell Go back to it's blurred image instead of the components we have right now. I was thinking if we could get some images of their features we could switch out the blurred coming soon with those new features (After we've structured it). Also again @Bookie0 feel free to add on/modify this and if either you @andrew032011 or @cchrischen! Have any ideas!
> 
> Also that sounds like a great Idea!

Awesome I think I might wait on the new design and address it within this PR!

**Comment by andrew032011** [comment]
> > > > Might need to reach out to Carriage to see if they can send us any designs we can use to update else we can just put cornell go back to the coming soon screen we had before
> > > 
> > > 
> > > Oh is this CornellGo or Carriage? If we need something feel free to tag the PM in one of their channels and cc me + PM’s + designers.
> > 
> > 
> > This is for Carriage. Right now Carriage has the Coming Soon/blurred image as a place holder and Clement suggested that we should keep it consistent and maybe revert the Cornell Go back to it's blurred image instead of the components we have right now. I was thinking if we could get some images of their features we could switch out the blurred coming soon with those new features (After we've structured it). Also again @Bookie0 feel free to add on/modify this and if either you @andrew032011 or @cchrischen! Have any ideas!
> > Also that sounds like a great Idea!
> 
> Awesome I think I might wait on the new design and address it within this PR! Thank you so much for doing that Andrew!

I think depending on when the design gets out we could merge this in first and take it as a follow-up PR! Just to keep the PR’s small and focused and new.cornelldti.org as close to launch ready as possible.

File: new-dti-website/components/products/FloatingImages.tsx (hunk 1 above)
Comment (Bookie0) [reply new-dti-website/components/products/FloatingImages.tsx:63]: sure! lmk when u'd like me to re-review :)

**Review by Bookie0 — APPROVED** [review]
hmm the images still seem to be moving despite the setting being on, like i [said here](https://github.com/cornell-dti/idol/pull/756#pullrequestreview-2473979566)

I know we've spent a lot of time on this, and since the animation is only so slight, I'm fine with signing off on this if we have to move on.

But just thing we can catch later on i think :)
https://github.com/user-attachments/assets/5674c280-0a5b-4c93-85c8-3bf7bd43f8ff

**Comment by JasonMun7** [comment]
> hmm the images still seem to be moving despite the setting being on, like i [said here](https://github.com/cornell-dti/idol/pull/756#discussion_r1864099204)
> 
> I know we've spent a lot of time on this, and since the animation is only so slight, I'm fine with signing off on this if we have to move on.
> 
> But just thing we can catch later on i think :) https://github.com/user-attachments/assets/5674c280-0a5b-4c93-85c8-3bf7bd43f8ff

Hmm I'm wondering if this might have to do with the hydration error as it seems to work fine on my local machine. I think i'll move on for now until we get through the pages/hydration error. Thanks @Bookie0 !

**Comment by Bookie0** [comment]
@JasonMun7 ok sounds good!

**Pull request merged**